lib/tessel/deployment/python.js

```diff
@@ -0,0 +1,151 @@
+// System Objects
+// var cp = require('child_process');
+var path = require('path');
+// var zlib = require('zlib');
+
+// Third Party Dependencies
+var fs = require('fs-extra');
+var fsTemp = require('fs-temp');
+var Ignore = require('fstream-ignore');
+// var minimatch = require('minimatch');
+// var request = require('request');
+var tags = require('common-tags');
+var tar = require('tar');
+// var uglify = require('uglify-js');
+// var urljoin = require('url-join');
+
+// Internal
+var lists = require('./lists/python');
+var glob = require('./glob');
+
+
+var exportables = {
+  meta: {
+    name: 'python',
+    extname: 'py',
+    binary: 'python',
+    entry: '__init__.py',
+    configuration: '__init__.py',
+    checkConfiguration: (pushdir, basename, program) => {
+      // var packageJson = fs.readJsonSync(path.join(pushdir, 'package.json'));
```

Comment:
No, I'd rather not. This here for "guidance" when we know better what the Python constraints will be.